.github/settings.yml

```diff
@@ -0,0 +1,35 @@
+repository:
+  name: httprouterpersist
+  description: null
+  homepage: null
+  topics: ''
+  private: false
+  has_issues: true
+  has_projects: true
+  has_wiki: true
+  has_downloads: true
+  default_branch: master
+  allow_squash_merge: true
+  allow_merge_commit: true
+  allow_rebase_merge: true
+  enable_automated_security_fixes: true
+  enable_vulnerability_alerts: true
+teams:
+  - name: developers
+    permission: push
+  - name: tech-ops-team
+    permission: admin
+  - name: web-team
+    permission: admin
+branches:
+  - name: '*-base'
```

Comment:
We can omit this for most repos. It's not relevant on tools like this that aren't under active development.

Reply:
Sorry, this came from the settings generator.  How about we simply switch this to `master`?

```suggestion
  - name: master
```